docs/howto/platform-sdk-quickstart-guide.md

```diff
@@ -0,0 +1,194 @@
+# Platform SDK Quickstart Guide
+
+#### Python SDK v2
+
+Version 2 of AYX Python SDK is now available. For the best experience,
+and to ensure compatibility with the most recent versions of Designer,
+please use v2 of [AYX Python
+SDK.](https://help.alteryx.com/developer-help/ayx-python-sdk-v2)
+
+-   To learn more about v2 of the AYX Python SDK, go to [Changes and
+    Improvements to Python SDK in
+    v2.0.0](https://help.alteryx.com/developer-help/changes-and-improvements-python-sdk-v2).
+-   If you need to access v1 documentation, you can [download the
+    content](https://help.alteryx.com/sites/default/files/2022-02/1.0_0.zip).
+
+The steps below illustrate how to use the latest developer tools to
+create a custom plugin or tool for use in Alteryx. For more information
+about new SDKs and their usage within Alteryx, please visit [Platform
+SDK](https://help.alteryx.com/developer-help/platform-sdk).
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version